### DISCUSSION TITLE
Fullscreen article on tablet and foldable

### DISCUSSION THREAD

**Discussion by alanvww** [jocmp/capyreader#499] [Feature Requests]
Hi there, I’d love to see a feature that lets me hide the sidebar and read articles full-screen on tablet and foldable devices. This would make for a more immersive reading experience.

**Comment by jocmp** [comment]
Hey @alanvww thanks for your support and for the suggestion! I definitely want to do this sometime soon as it's come up before. There are a few things I need to try to get there, but I'll keep this thread updated with any progress.

**Reply by alanvww** [reply]
Thanks for your wonderful project! I have recently switched back to Android. Capy Reader definitely makes my life easier by keeping my reading routine enjoyable, and I know maintaining an open source project is hard. Take your time and keep up the good work!

**Comment by jocmp** [comment]
just a small update. I was able to get this working as an option. There's a few more cosmetic touch-ups I want to do before go-live, but here's what it looks like.

## Before

<img src="https://github.com/user-attachments/assets/b334d351-8879-42ef-98bd-d64f76a1019f" width="500px">

## After

<img src="https://github.com/user-attachments/assets/5e21b249-eeed-4609-8725-c6745f89516d" width="500px">

**Reply by jocmp** [reply]
The first pass at this is available in [2024.12.1081-dev](https://github.com/jocmp/capyreader/releases/tag/2024.12.1081-dev). The setting is under Display & Appearance > Preferred Layout > Single Column.

There's a lot more I want to do with the tablet mode that I'll [track separately](https://github.com/jocmp/capyreader/issues/583). Let me know what you think!

**Reply by alanvww** [reply]
@jocmp love the full screen article view! Just tested on my Pixel 9 Fold, it works well. And something will be very great to see: while the article now can show fullscreen, can we make the feed list and article list show together? ![Screenshot_20241206-203730.png](https://github.com/user-attachments/assets/5a2761e1-2da5-4952-8745-dcbbdf46ac00)

**Reply by jocmp** [reply]
hey thanks for trying it out. I'm glad you mentioned the combined lists - that's another thing I want to do as well. I'll add it to the list.

**Reply by jocmp** [reply]
The combined lists aren't quite ready, but it's close! Here's how it works.

When the reader layout is set to "single" the feed and article lists are shown together. At the 9s mark in the video, the Fold emulator is closed showing the compact view, and then at 18s it's reopened to show the combined list layout again.

https://github.com/user-attachments/assets/f6aa8da5-92d9-4e36-bae2-b38723fbaaea

**Reply by jocmp** [reply]
These changes are live in [2024.12.1088-dev](https://github.com/jocmp/capyreader/releases/tag/2024.12.1088-dev).
